#Массив значений статусов ключей системы, 0 - выкл. 1 - вкл.
uint8[16] status